### PR TITLE
Added Promela implementation

### PR DESCRIPTION
**Pull request by tklaback**
This branch adds the promela visitor implementation which generates the promela from the bpmn and state.

### CODE REVIEW

File: src/bpmncwpverify/bpmn.py [hunk 1]
```diff
@@ -114,6 +115,12 @@ def accept(self, visitor: "BpmnVisitor") -> None:
         self.traverse_outflows_if_result(visitor, result)
         visitor.end_visit_task(self)
 
+    def cleanup(self) -> None:
```
Comment (ericmercer) [inline src/bpmncwpverify/bpmn.py:118]: Wasn't this function going away?

File: src/bpmncwpverify/visitors.py [hunk 2]
```diff
@@ -99,25 +111,339 @@ def end_visit_bpmn(self, bpmn: Bpmn) -> None:
...
+                put_flow_ids,
+                element_id,
+                option_type,
+            )
+        )
+
+    def gen_behavior_model(self, place_name: str, update_state: bool = False) -> str:
+        ret = "//{}\n".format(place_name)
+        ret += "inline {x}_BehaviorModel(){{\n".format(x=place_name)
+        # TODO: figure out how to do this without modifies file
```
Comment (ericmercer) [inline src/bpmncwpverify/visitors.py:338]: I know now how we are going to inject the behavior model (mostly). I'm going to use the out-going edge from a task to assert the `post-condition` that must hold leaving the task. We can generate from that the Promela code. The only think we need to figure out is how we want to specify non-determinism. I'm thinking of something like x in {1, 2, 3} type notation saying that x has one of the values in the set.

**Review by ericmercer — COMMENTED** [review]
There are a few comments. But in general, feel free to commit. We'll talk more about the behavior model.

File: src/bpmncwpverify/bpmn.py (hunk 1 above)
Comment (tklaback) [reply src/bpmncwpverify/bpmn.py:118]: yes, I must have missed it while solving merge conflict, I will take care of this.

File: src/bpmncwpverify/visitors.py (hunk 2 above)
Comment (tklaback) [reply src/bpmncwpverify/visitors.py:338]: ah, yes, this makes sense

**Pull request merged**